readme.md

```diff
@@ -522,3 +522,4 @@ Some users may want to extend Zustand's feature set which can be done using thir
 ## Comparison with other libraries
 
 - [Difference between zustand and valtio](https://github.com/pmndrs/zustand/wiki/Difference-between-zustand-and-valtio)
```

Comment:
btw, we should fix this link. Can anyone open a PR?

Reply:
fixed